wiki/content/deploy/tls-configuration.md

```diff
@@ -167,7 +206,7 @@ $ dgraph cert -n localhost -c dgraphuser
 $ dgraph alpha --tls_dir tls --tls_client_auth="REQUIREANDVERIFY"
```

Comment:
tls_internal_port_enabled=true flag is required to enable internal ports encryption